### DISCUSSION TITLE
How to spawn new vehicles from the in-ramp?

### DISCUSSION THREAD

**Discussion by rjingHD** [metadriverse/metadrive#723] [Q&A] [answered]
Hey people, I would like to know how can we generate new vehicles from the starting point of the side road of the in-ramp in the multiagent setting?

**Comment by QuanyiLi** [comment, marked as answer]
Hi @rjingHD 

First, you need to find the lane ID of the lane that you are interested in. So you can place a vehicle on the lane based on the longitude and lateral of the lane coordinates. Please refer to:

https://github.com/metadriverse/metadrive/issues/731
